src/actions.mli

```diff
@@ -0,0 +1,88 @@
+val job_action :
+     bot_info:Bot_components.Bot_info.t
+  -> Bot_components.GitLab_types.job_info
+  -> github_of_gitlab:(string -> string option)
+  -> unit Lwt.t
+
+val pipeline_action :
+     bot_info:Bot_components.Bot_info.t
+  -> Bot_components.GitLab_types.pipeline_info
+  -> github_of_gitlab:(string -> string option)
+  -> unit Lwt.t
+
+val coq_bug_minimizer_results_action :
+     bot_info:Bot_components.Bot_info.t
+  -> string
+  -> (Cohttp.Response.t * Cohttp_lwt__Body.t) Lwt.t
+
+val merge_pull_request :
+     bot_info:Bot_components.Bot_info.t
+  -> comment_info:Bot_components.GitHub_types.comment_info
+  -> unit Lwt.t
+
+val pull_request_updated :
```

Comment:
If I'm not mistaken, `pull_request_updated` is not used outside this module so should not be exported (the function that is called from outside being `pull_request_updated_action` or `run_ci`).